Tools/ardupilotwaf/boards.py

```diff
@@ -261,12 +269,26 @@ def configure_env(self, cfg, env):
             'SITL',
         ]
 
+        env.GIT_SUBMODULES += [
+            'uavcan',
```

Comment:
I think it should check for SLCAN or whatever is the name for the linux implementation of CAN bus !